File: conference/management-bc/web/src/main/java/com/microsoft/conference/management/domain/Events/SeatTypeQuantityChanged.cs
using System;
using ENode.Eventing;

namespace ConferenceManagement
{
    public class SeatTypeQuantityChanged : DomainEvent<Guid>
    {
        public Guid SeatTypeId { get; private set; }
        public int Quantity { get; private set; }
        public int AvailableQuantity { get; private set; }

        public SeatTypeQuantityChanged() { }
        public SeatTypeQuantityChanged(Guid seatTypeId, int quantity, int availableQuantity)
        {
            SeatTypeId = seatTypeId;
            Quantity = quantity;
            AvailableQuantity = availableQuantity;
        }
    }
}

File: samples/conference/management-bc/src/main/java/com/microsoft/conference/management/domain/Events/SeatTypeQuantityChanged.java
package com.microsoft.conference.management.domain.Events;

import org.enodeframework.eventing.DomainEvent;

public class SeatTypeQuantityChanged extends DomainEvent<String> {
    public String SeatTypeId;
    public int Quantity;
    public int AvailableQuantity;

    public SeatTypeQuantityChanged() {
    }

    public SeatTypeQuantityChanged(String seatTypeId, int quantity, int availableQuantity) {
        SeatTypeId = seatTypeId;
        Quantity = quantity;
        AvailableQuantity = availableQuantity;
    }
}
